cvat-core/src/api-implementation.ts

```diff
@@ -106,6 +106,28 @@ export default function implementAPI(cvat) {
         await serverProxy.server.resetPassword(newPassword1, newPassword2, uid, token);
     };
 
+    cvat.server.acceptInvitation.implementation = async (
+        username,
+        firstName,
+        lastName,
+        email,
+        password,
+        userConfirmations,
+        key,
+    ) => {
+        const orgSlug = await serverProxy.server.acceptInvitation(
+            username,
+            firstName,
+            lastName,
+            email,
+            password,
+            userConfirmations,
+            key,
+        );
+
+        return orgSlug;
+    };
```

Comment:
Not sure that ``server`` is good namespace here.
What do you think about ``cvat.organizations.acceptInvitation``?